docs/en/revocation-lists.rst

```diff
@@ -190,18 +198,25 @@ The requests to the *Issuer Revocation endpoint* MUST be HTTP with method POST,
     * - **Claim**
       - **Description**
       - **Reference**
-    * - **credential_pop**
-      - It MUST contain a JWT proof of possession of the cryptographic key the Credential to be revoked shall be bound to. See Section :ref:`Credential Proof of Possession <sec_revocation_credential_pop>` for more details. 
-      - `[OAuth Status Attestation draft 01] <https://datatracker.ietf.org/doc/draft-demarco-status-attestations/01/>`_
+    * - **revocation_assertion_requests**
```

Comment:
this parameter name must be changed

the scope of the request is to request the revocation of credentials or wallet instance attestations

Reply:
the parameter name is changed with revocation_requests